omniscidb/QueryEngine/CalciteDeserializerUtils.h

```diff
@@ -74,6 +74,18 @@ inline hdk::ir::OpType to_sql_op(const std::string& op_str) {
   if (op_str == std::string("NOT")) {
     return hdk::ir::OpType::kNot;
   }
+  if (op_str == std::string("BW_AND")) {
+    return hdk::ir::OpType::kBwAnd;
+  }
+  if (op_str == std::string("BW_OR")) {
+    return hdk::ir::OpType::kBwOr;
+  }
+  if (op_str == std::string("BW_XOR")) {
+    return hdk::ir::OpType::kBwXor;
+  }
+  if (op_str == std::string("BW_NOT")) {
+    return hdk::ir::OpType::kBwNot;
+  }
```

Comment:
Calcite has support for BIT_AND/OR/XOR. Support for BIT_NOT is added here - https://github.com/intel-ai/hdk/pull/580. To be compatible with Calcite, the operation name prefix should be changed to `BIT_`.

Reply:
Added both options